### ISSUE TITLE
Oscilloscope pass/fail mask and test

### ISSUE THREAD

**Issue by Diti24** [python-ivi/python-ivi#48]
Hi,
I am integrating a new oscilloscope into this library.
I was not able to find any functionality for the Mask pass/fail tests.
I am refering to http://cdn.rohde-schwarz.com/pws/dl_downloads/dl_common_library/dl_manuals/gb_1/h/hmo1002_1202/HMO1002_1202_SCPI_ProgrammersManual_en_01.pdf
Chapter 2.8 page 117.

Could some one give me some tipps? how to proceed?
Thanks

**Comment by alexforencich** [comment]
Yeah, right now a lot of that high-level functionality (protocol decoding, mask testing, etc.) doesn't have nice abstractions.  It's on the to-do list.
